jimmy-nic-danul/README.md

```diff
@@ -0,0 +1,22 @@
+# Templating
+
+**Author**: Danul, Dominic, Jimmy
+**Version**: 1.1.1
+
+## Overview
+Utilizing handlebars to template this week's labs for the articles portion.
+
+## Getting Started
+The user must template the HTML inside the script tag placed in the index.html. They will also have to compile the script using Handlebars and use the articles object to populate it.
+
+## Architecture
+We will use Handlebars.js, HTML, JS, and CSS.
+
+## Change Log
```

Comment:
make sure you are previewing your markdown files. These lines couldd to be a list with `- ` at the beginning of each line. Right now they are all jumbling together on 1 line.